runtime/queries/purescript/highlights.scm

```diff
@@ -41,18 +25,19 @@
    "]"
  ] @punctuation.bracket
 
- [
- (comma)
- ";"
- ] @punctuation.delimiter
+ (comma) @punctuation.delimiter
```

Comment:
Removed `;` is also off-topic but it's not a token we parse in the upstream grammar. It was present here because this is a fork of Haskell's grammar, and I forgot to remove it previously.